### ISSUE TITLE
Transformasjon-konverteringer

### ISSUE THREAD

**Issue by helemork** [Artsdatabanken/assessments-frontend#101]
Ting som bør skje på transformasjonsnivå, ikke i frontend:

- [ ] for alle LC, NA og NE-arter bør region være tom/returnere state "2" på alle element. Altså ikke satt. Enhver data som er satt der kan bli potensielt ansett som feil da det ikke var krav om revidering av potensiell tidligere innlagt data der, så den kan være foreldet.
- [ ] Svalbard/Norge osv ut av artsgruppe og ekspertkomite
- [ ] For ExpertStatement burde stripping av HTML og korrigering av inline html utføres på høyere nivå, da også data levert i API med fordel kunne sett ok ut. Nåværende meget tungvindte men fungerende versjon ligger i partial _oppsumer, https://github.com/Artsdatabanken/assessments-frontend/blob/master/Assessments.Frontend.Web/Views/Test/partials_2021/_Oppsumer.cshtml <-- P.S. rekkefølgen må følges da flere typer html først strippes og tømmes før de ender opp i en type som til slutt blir en overskrift.

**Comment by eirikn-artsdatabanken** [comment]
@helemork Jeg finner ikke noe i modellen som heter artsgruppe. ekspertkomite (ExpertCommittee) har allerede kode som tar bort vurderingsområde: https://github.com/Artsdatabanken/assessments-frontend/blob/db06033a0207c5bf612bf5d4496c8cf9662f3241/Assessments.Mapping/SpeciesAssessment2021Profile.cs#L225

**Comment by helemork** [comment]
Som nevnt på forrige møte, @eirikn-artsdatabanken  var det ønskelig med artsgruppe som nytt felt. 
Mer spesifikt: 
**Det vi har:**
- ekspertgruppe = ekspertgruppe formatert uten svalbard/norge, med transformasjon som splitter opp artsgruppene
**Det vi vil ha:**
- ekspertgruppe = ekspertgruppe formatert uten svalbard/norge
- artsgruppe = ekspertgruppe formatert uten svalbard/norge, med transformasjon som splitter opp artsgruppene

**Comment by eirikn-artsdatabanken** [comment]
artsgruppe skal splittes før transformasjonen her https://github.com/Artsdatabanken/assessments-frontend/blob/db06033a0207c5bf612bf5d4496c8cf9662f3241/Assessments.Mapping/SpeciesAssessment2021Profile.cs#L185

**Comment by helemork** [comment]
artsgruppe skal være akkuratt det ekspertgruppe er nå :)

**Comment by eirikn-artsdatabanken** [comment]
ok, ser ut som det bare er en ekspertgruppe som har mere enn et navn: "Amfibier, reptiler"

**Comment by helemork** [comment]
Det stemmer ikke. 
Se https://artsdatabanken.no/Files/27117/Ekspertkomit_er_og_medlemmer_i_2015 for 2015-komiteene, det skal være noen få, små endringer til 2021 på gruppesammensetningene, men i stor del de samme.

**Comment by eirikn-artsdatabanken** [comment]
her er resultatene etter transformering og hentet fra databasen (rødliste 2021) gruppert på ekspertkomiteene:
https://assessments-fe.test.artsdatabanken.no/api/Species2021?$count=true&$apply=groupby((ExpertCommittee))

det er nok de vi må forholde oss til

**Comment by KristineLund** [comment]
Jeg har ingen sterke meninger om hvor i prosessen det er best å legge til SpeciesGroup @eirikn-artsdatabanken - men ville tro det var best å gjøre det tidlig så "artsgruppe" og "ekspertgruppe" betyr det samme gjennom hele løpet :)

**Comment by helemork** [comment]
Ja, dette er en duplikat av #90, hadde ikke sett den blitt satt opp.
Det som kom fra databasen på ekspertkomite var ikke splittet for noen dager siden, det er det transformasjonen du la til igjen gjør :) Så ved å kun bruke den på det koperte feltet speciesgroup bør vi komme i mål

**Comment by steinho** [comment]
> Ting som bør skje på transformasjonsnivå, ikke i frontend:
> 
> * [ ]  for alle LC, NA og NE-arter bør region være tom/returnere state "2" på alle element. Altså ikke satt. Enhver data som er satt der kan bli potensielt ansett som feil da det ikke var krav om revidering av potensiell tidligere innlagt data der, så den kan være foreldet.

Har fixet dette lokalt nå - kommer i ett innsjekk

**Comment by steinho** [comment]
Usikker på om det gjenstår noe annet på denne?

**Comment by helemork** [comment]
Nei, det burde lukke denne. Link gjerne inn denne tråden når du setter opp pullrequest, så lukker jeg den når jeg ser over :)
